README.md

```diff
@@ -74,7 +74,8 @@ We use the div to control the entire height and width, and the span to get acces
 	* `columnClass`: a CSS class to add to the header cell and the column cell
 	* `cellClass`: a CSS class to add to each cell in this column (except for the header) - added to the <span>
 	* `wideCellClass`: a CSS class to add to each cell in this column (except for the header) - added to the <div>
-	* `value`: the attribute on the node to use as the value for this cell - entered as the <span> text. Can be a string or a function.
+	* `value`: the attribute on the node to use as the value for this cell - entered as the <span> text. Must be a string.
```

Comment:
> Must be a string

Not really. It also can handle integers or other numbers, or even booleans, as you showed in the example. Probably should say "Must be a string, number, boolean or other primitive."